dask_cuda/local_cuda_cluster.py

```diff
@@ -288,7 +288,9 @@ def __init__(
 
         data = kwargs.pop("data", None)
         if data is None:
-            if self.jit_unspill:
+            if device_memory_limit is None and memory_limit is None:
+                data = {}
+            elif self.jit_unspill:
```

Comment:
FWIW, and you're not changing this code, `self.jit_unspill` is a property that is only ever read in `__init__` so it could presumably not be set at all and you could just use `dask.config.get("jit-unspill", default=jit_unspill)`

Reply:
Do you want to do that or shall I?

Reply:
If you have time, please do :)